src/chainparams.cpp

```diff
@@ -171,7 +171,6 @@ class CTestNetParams : public CMainParams {
         vAlertPubKey = ParseHex("044e7a1553392325c871c5ace5d6ad73501c66f4c185d6b0453cf45dec5a1322e705c672ac1a27ef7cdaf588c10effdf50ed5f95f85f2f54a5f6159fca394ed0c6");
         nDefaultPort = 18233;
         nMinerThreads = 0;
-        nMaxTipAge = 0x7fffffff;
```

Comment:
We could add it back in later (e.g. in 1.0.1). Meanwhile, it's simple enough to work around.